### DISCUSSION TITLE
Looking for I4R4 version of Vortex_to_D3plot converter.

### DISCUSSION THREAD

**Discussion by nickmeng2017** [OpenRadioss/OpenRadioss#2593] [Contributors]
The Vortex_to_D3plot converter generates I8R8 file.  Is there any way to generate I4R4 file? 

Thanks,
Nick

**Comment by Vortex-CAE** [comment]
Hi @nickmeng2017,

If you go into Anim_to_D3plot.py and change the following lines:

```
self._d3plot.header.itype = np.int64
self._d3plot.header.ftype = np.float64
self._d3plot.header.wordsize = 8 
```

to:

```
self._d3plot.header.itype = np.int32
self._d3plot.header.ftype = np.float32
self._d3plot.header.wordsize = 4
```

I think that will do what you are asking for, please let me know if this is not the case.

Kind regards,
David

**Reply by nickmeng2017** [reply]
Thanks a lot! It works.  One more question.  Is it possible to dump the header of A file in ASCII format with the python code?

**Reply by Vortex-CAE** [reply]
There isn't an ASCII equivalent for D3plot like there is for Binout. You can use the reader to generate the dictionary of arrays, but you would need to write some code to write that output to a text file, there are a number of other Python libraries that can simplify that process such as csv.

Hope that helps,
David

**Comment by nickmeng2017** [comment]
Thanks a lot for the information.

It works.  One more question.   Is it possible to dump the binary file into ASCII file ?

Thanks,
Nick
